libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_DNS.c

```diff
@@ -975,11 +975,11 @@ uint32_t ulIPAddress = 0uL;
 uint8_t *pucByte;
 size_t uxSourceBytesRemaining;
 uint16_t x, usDataLength, usQuestions;
-BaseType_t xDoStore = xExpected;
 #if( ipconfigUSE_LLMNR == 1 )
 	uint16_t usType = 0, usClass = 0;
 #endif
 #if( ipconfigUSE_DNS_CACHE == 1 ) || ( ipconfigDNS_USE_CALLBACKS == 1 )
+	BaseType_t xDoStore = xExpected;
```

Comment:
Does this change compile regardless of the combination of these #defines set?

Reply:
I have verified that this variable is only used in the sections of the function definition guarded by `ipconfigUSE_DNS_CACHE` or `ipconfigDNS_USE_CALLBACK`
https://github.com/aws/amazon-freertos/blob/abff7f3604709c9407cf373f09986a87c75d4001/libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_DNS.c#L1108-L1134